src/screens/discover/components/DiscoverSearchInput.tsx

```diff
@@ -164,10 +184,19 @@ const ExchangeSearch = (
     };
   });
 
+  const searchInputValue = useAnimatedProps(() => {
+    // Removing the value when the input is focused allows the input to be reset to the correct value on blur
```

Comment:
I don't understand this comment, doesn't seem related to the rest?

Reply:
It's similar to what we do for DappBrowser and Swaps search which let's us use an uncontrolled input in a way that feels controlled.